styles/themes/_light.scss

```diff
@@ -114,6 +114,14 @@ html.light-mode {
         color: $k-hovergrey !important;
       }
     }
+    .navbar__sign-out-button {
+      border-color: $black !important;
+      box-shadow: 4px 4px 0px $black !important;
+      color: $black;
+      &:hover {
+        background-color: $k-accentlight !important;
+      }
+    }
```

Comment:
since we'll use `NeoButton`, is it stil needed?

Reply:
> Are you sure there is no css style left to remove?
> 
>  Enregistrement.de.l.ecran.2023-01-05.a.8.46.44.AM.mov

accidentally pushed, I'm not done, sorry